### PR TITLE
fix(TDI-38848): tAzureStorageQueueOutput performance enhancement

### PR DESCRIPTION
**Pull request by akhabali**
**What is the current behavior?** (You can also link to an open issue here)
https://jira.talendforge.org/browse/TDI-38848

**What is the new behavior?**
We use parallelism while sending message to Azure queue to enhance the performance of the component.
this increase the performance form 10 msg/sec to 120 msg/sec at least. 

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/392/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/392/)

**Comment by gonchar-ivan** [comment]
Can be merged after small fix

File: components/components-azurestorage/src/main/java/org/talend/components/azurestorage/queue/runtime/QueueMessage.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+// ============================================================================
+//
+// Copyright (C) 2006-2016 Talend Inc. - www.talend.com
```
Comment (akhabali) [inline components/components-azurestorage/src/main/java/org/talend/components/azurestorage/queue/runtime/QueueMessage.java]: this should be automated in the codetemplates i will fixed there and make a PR

Comment (akhabali) [inline components/components-azurestorage/src/main/java/org/talend/components/azurestorage/queue/runtime/QueueMessage.java]: https://github.com/Talend/components/pull/757

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/400/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/400/)

**Pull request merged**